src/command_modules/azure-cli-sql/tests/test_sql_commands.py

```diff
@@ -865,3 +866,84 @@ def test_sql_elastic_pools_mgmt(self, resource_group, resource_group_location, s
+        self.cmd('sql server firewall-rule create --name {} -g {} --server {} '
+                 '--start-ip-address {} --end-ip-address {}'
+                 .format(firewall_rule_1, rg, server,
+                         start_ip_address_1, end_ip_address_1),
+                 checks=[
+                     JMESPathCheck('name', firewall_rule_1),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('startIpAddress', start_ip_address_1),
+                     JMESPathCheck('endIpAddress', end_ip_address_1)])
+
+        # create db
+        self.cmd('sql db create -g {} --server {} --name {}'
+                 .format(rg, server, db_name),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', db_name),
+                     JMESPathCheck('location', loc_long),
+                     JMESPathCheck('elasticPoolName', None),
+                     JMESPathCheck('status', 'Online')])
+
+        self.cmd('sql db create -g {} --server {} --name {}'
+                 .format(rg, server, db_name2),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', db_name2),
+                     JMESPathCheck('location', loc_long),
+                     JMESPathCheck('elasticPoolName', None),
+                     JMESPathCheck('status', 'Online')])
+
+        # Backup to new dacpac
+        # get storage account endpoint
+        storage_endpoint = self.cmd('storage account show -g {} -n {}'
+                                    ' --query primaryEndpoints.blob'
+                                    .format(rg, storage_account)).get_output_in_json()
+
+        # get storage account key
+        key = self.cmd('storage account keys list -g {} -n {} --query [0].value'
+                       .format(rg, storage_account)).get_output_in_json()
+
+        # create storage account blob container
+        self.cmd('storage container create -n {} --account-name {} --account-key {} '
+                 .format(container, sa, key),
+                 checks=[
+                     JMESPathCheck('created', True)])
+
+        # export database to blob container
+        self.cmd('sql db export -s {} -n {} -g {} -p {} -u {}'
+                 ' --storage-key {} --storage-key-type StorageAccessKey'
+                 ' --storage-uri {}{}/testbacpac.bacpac'
+                 .format(server, db_name, rg, admin_password, admin_login, key,
+                         storage_endpoint, container))
+
+        # import bacpac to second database
+        self.cmd('sql db import -s {} -n {} -g {} -p {} -u {}'
+                 ' --storage-key {} --storage-key-type StorageAccessKey'
+                 ' --storage-uri {}{}/testbacpac.bacpac'
+                 .format(server, db_name2, rg, admin_password, admin_login, key,
+                         storage_endpoint, container))
```

Comment:
There aren't any assertion on the last two command execution except the exit code. Shall we add more assertions?